.circleci/config.yml

```diff
@@ -190,3 +309,7 @@ workflows:
       - unit-test:
           context:
           - docker-hub-creds
+
```

Comment:
Whitespace between `test` and `windows-test` but not between `version: 2` and `test`. I'm OK with either, but let's be consistent.